ksql-engine/src/test/java/io/confluent/ksql/integration/SecureIntegrationTest.java

```diff
@@ -153,6 +152,7 @@ public void after() {
     }
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
I added this so that I can get a green build - will remove before merging (separate PR to fix this)